blueprints/gitlab-ci-phantom/files/gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+image: node:latest
+
+before_script:
+  - npm config set spin false
+  - npm install -g bower
+  - npm install -g ember-cli
+  - npm install -g phantomjs-prebuilt
+  - npm install
+  - bower install --allow-root
+
+test:
+  script:
+    - ember test
```

Comment:
we should probably use `npm test` here to remove the need above to install the `ember-cli` package globally.